dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -432,6 +441,14 @@ def color_for_status(status)
   element_value_is(selector, expected_value)
 end
 
+Then /^element "([^"]*)" has escaped value "([^"]*)"$/ do |selector, expected_value|
+  element_value_is(selector, YAML.load(%Q(---\n"#{expected_value}"\n)))
+end
+
+Then /^element "([^"]*)" has escaped value '([^']*)'$/ do |selector, expected_value|
+  element_value_is(selector, YAML.load(%Q(---\n"#{expected_value.gsub('"', '\"')}"\n)))
+end
```

Comment:
I feel like this can be combined through regex magic. But its okay to leave it for readability